assets/certs/ca.config

```diff
@@ -0,0 +1,10 @@
+[req]
+distinguished_name = req_distinguished_name
+prompt = no
+[req_distinguished_name]
+C = US
+ST = California
+L = San Francisco
+O = "Honest Achmed's Used Certificates"
+OU = Hastily-Generated Values Divison
```

Comment:
```suggestion
OU = Hastily-Generated Values Division
```

Reply:
This was pulled off the existing certificate.  I can update it but that'd also churn the CSR, which I left untouched.

Reply:
No worries. I just notice these things easily. :-)